models/intel/smartlab-object-detection-0001/accuracy-check.yml

```diff
@@ -0,0 +1,35 @@
+models:
+  - name: smartlab-object-detection-0001
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: yolox
+          anchors: yolox
+          classes: 10
+
+    datasets:
+      - name: DSI1867
+        data_source: ..\..\..\DSI-1867\object_detection\streams_1\top\images
```

Comment:
I mean differnet when says relative path.
AC has command line arguments -s, --source which stores the root of datasets directory. Path in configs should be written relative to it without and special symbols like `..`, `.`, `~`

Reply:
beside that data_source, annotation, annotation_conversion should be moved to dataset_definitions file, they should not be defined in model config
https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_definitions.yml

Reply:
fixed